packages/react-storage/src/components/StorageBrowser/context/elements/definitions.ts

```diff
@@ -25,7 +25,7 @@ export interface StorageBrowserElements {
 }
 
 export type PaginateVariant = `paginate-${'next' | 'current' | 'previous'}`;
-
+export type MessageVariant = 'info' | 'success' | 'error' | 'warning';
```

Comment:
What is your opinion on scoping control specific `Variant` types this way? Added `PaginateVariant` to narrow the allowed `variant` values in `PaginateControl` but not sure it makes sense